contactsummary/js/src/contact-summary.js

```diff
@@ -6,7 +6,5 @@ require.config({
 });
 
 require(['contact-summary/app'], function () {
-    document.addEventListener('contactsummaryLoad', function () {
```

Comment:
after set the "contact summary" tab to be the first and the default tab instead of "personal details" tab the tab content don't get loaded and that's because the line below do not get called :

` angular.bootstrap(document.getElementById('contactsummary'), ['contactsummary']);`